homeassistant/components/device_tracker/automatic.py

```diff
@@ -38,65 +44,183 @@
 
 EVENT_AUTOMATIC_UPDATE = 'automatic_update'
 
+AUTOMATIC_CONFIG_FILE = 'automatic/session-{}.json'
```

Comment:
Can you prefix the directory with `.` so the user won't see it? (as they shouldn't edit it)